### DISCUSSION TITLE
Authorize value?

### DISCUSSION THREAD

**Discussion by annmc** [amberelectric/public-api#11] [Q&A] [answered]
Sorry for such a basic question, but what do I type in to the Value field to Authorize? 

I have generated a token (Amberapi) & I've tried substituting lots of different things in the format of apiKey  (http, Bearer) but keep getting "User is not authorized to access this resource with an explicit deny"

**Comment by madpilot** [comment, marked as answer]
You need to enter the string that starts with `psk_` (It'll disappear after you refresh the page - if you didn't copy it, just generate a new one)

**Reply by annmc** [reply]
Thanks, that worked. Obvious now that I know!
